pkgs/misc/vencord/default.nix

```diff
@@ -50,6 +50,11 @@ buildNpmPackage rec {
     ./disable-updater-ui.patch
   ];
 
+  postPatch = ''
+    # Prevent build scripts from including this file in builds
+    rm src/plugins/_core/settings.tsx.orig
```

Comment:
Do you know why it's being created in the first place?
It seems there's also `./scripts/build/common.mjs.orig`
And there are no `./src/utils/updater.ts.orig` and `./src/components/VencordSettings/UpdaterTab.tsx.orig`.

Reply:
It's possible that the patch doesn't apply cleanly (line numbers might have changed) which is why patch might leave the original file there as a backup.

I could try updating the patches to see if that fixes it

Reply:
That was indeed the case!